src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,10 @@
+$start-day: 606px;
```

Comment:
The variable `$start-day` is defined with a value of `606px`, which seems unusual for a variable named `start-day`. This name suggests it might represent a day of the week rather than a pixel value. Consider renaming this variable to better reflect its purpose or ensure that its usage aligns with its name.